scripts/create-tables.sql

```diff
@@ -1,3 +1,25 @@
+CREATE TABLE IF NOT EXISTS "users"
+(
+  id bigserial NOT NULL,
+  email varchar NOT NULL UNIQUE,
+  username varchar NOT NULL UNIQUE,
```

Comment:
I guess it's probably in here vs the migration file above, but same applies: index columns you search on.